vector/src/main/java/im/vector/adapters/VectorMediasViewerAdapter.java

```diff
@@ -641,65 +643,67 @@ private void playVideo(View pageView, VideoView videoView, File videoFile, Strin
     /**
      * Download the current media file, and export it to the Download folder of the device
      */
-    private void downloadMediaAndExportToDownloads() {
-        final SlidableMediaInfo mediaInfo = mMediasMessagesList.get(mLatestPrimaryItemPosition);
+    public void downloadMediaAndExportToDownloads() {
```

Comment:
Maybe take advantage of this refactoring to put this method elsewhere ?

Reply:
I thought about it, because it has nothing to do in an Adapter, but I cannot refactor the entire app on each PR ;)